src/workerd/api/actor-state.c++

```diff
@@ -889,8 +889,14 @@ void DurableObjectState::abort(jsg::Optional<kj::String> reason) {
     s.get()->getActorCacheInterface().shutdown(error);
   }
 
-  IoContext::current().abort(kj::cp(error));
-  kj::throwFatalException(kj::mv(error));
+  IoContext::current().abort(kj::mv(error));
+  js.terminateExecution();
```

Comment:
Yeah something I noticed with testing was that you could _catch_ the exception thrown from a call to `DurableObjectState::abort` in JS and do other stuff, but it didn't really seem useful or that terrible of a bug since it broke the output gate which meant you couldn't communicate to the outside world anyway.